src/model/guild/partial_guild.rs

```diff
@@ -934,6 +934,43 @@ impl PartialGuild {
         )
     }
 
+    /// Gets the highest role a [`Member`] of this Guild has.
+    ///
+    /// Returns None if the member has no roles or the member from this guild.
+    #[must_use]
+    pub fn member_highest_role(&self, member: &Member) -> Option<&Role> {
+        Guild::_member_highest_role_in(&self.roles, member)
+    }
+
+    /// Returns which of two [`User`]s has a higher [`Member`] hierarchy.
+    ///
+    /// Hierarchy is essentially who has the [`Role`] with the highest [`position`].
+    ///
+    /// Returns [`None`] if at least one of the given users' member instances is not present.
+    /// Returns [`None`] if the users have the same hierarchy, as neither are greater than the
+    /// other.
+    ///
+    /// If both user IDs are the same, [`None`] is returned. If one of the users is the guild
+    /// owner, their ID is returned.
+    ///
+    /// [`position`]: Role::position
```

Comment:
Please just link to the `Guild::greater_member_hierarchy` for the docs on this, I'm on the process of deduplicating documentation because it causes them to get out of sync when copied like this.

Reply:
fixed